rule_packages/cpp/Pointers.json

```diff
@@ -86,7 +86,8 @@
           "name": "A pointer to member virtual function shall only be tested for equality with null-pointer-constant",
           "precision": "very-high",
           "severity": "error",
-          "short_name": "PointerToMemberVirtualFunctionWithNullPointerConstant",
```

Comment:
I don't think we should change the `short_name` of any query we've already shipped. This is because the `short_name` affects the query id, and changing the `query_id` would have negative consequences, such as:

 * Invalidating existing deviation records, which reference query ids.
 * Invalidating query suites which reference queries explicitly.
 * Causing Code Scanning to treat results as "new" - because it uses the query id as part of the logical identity for a query. Existing alerts for these queries would be closed, and new alerts opened. If users had closed alerts as false positives, that determination would be lost - because the alerts would reopened a fresh.

Noting that there are a number of rules where this is the case, not just this one, and we should address them all.